### PR TITLE
Testing/invite actions

### PR DESCRIPTION
**Pull request by jon-schlandt**
## Is this a fix or a feature?
This is a feature:
  - The tests for the dashboard have now been completed; accepting, declining, and canceling (both player and host) are now covered
  - Will need to test to ensure the 'no events found' message appear once its implementation is fixed

## What is the change?
More testing has been added to the dashboard.

## Where should the reviewer start?
With the dashboard testing spec.

## How should this be tested?
Via Cypress.

### CODE REVIEW

File: cypress/integration/dashboard.spec.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 describe('When a user first accesses the Dashboard', () => {
   beforeEach('setup stubs and visit Dashboard', () => {
-    cy.setDataStubs()
+    cy.setReadStubs()
```
Comment (errabun) [inline cypress/integration/dashboard.spec.js:3]: I like how you made this into a function call with the stubs, helps with the readability!

File: cypress/support/commands.js [hunk 2]
```diff
@@ -24,15 +24,38 @@
...
+const devEnv = 'http://861341e035fa.ngrok.io'
 
-Cypress.Commands.add('setDataStubs', () => {
-  cy.fixture('../fixtures/players.json')
-    .then(players => cy.intercept(`${devEnv}api/v1/players`, players))
-  
-  cy.fixture('../fixtures/courses.json')
-    .then(courses => cy.intercept(`${devEnv}api/v1/courses`, courses))
+Cypress.Commands.add('setReadStubs', () => {
+  cy.intercept(
```
Comment (errabun) [inline cypress/support/commands.js:30]: This is the first time I've seen Cypress.Commands...nice!

**Review by errabun — APPROVED** [review]

File: src/components/TeeTimeContainer/TeeTimeContainer.js [hunk 3]
```diff
@@ -73,6 +73,8 @@ const TeeTimeContainer = ({
 
   }, [events, getEventType]);
 
+  console.log(events)
```
Comment (tysnj) [inline src/components/TeeTimeContainer/TeeTimeContainer.js:76]: we'll want to circle back and remove the console.log

**Review by tysnj — APPROVED** [review]
Code looks good and tests are passing on review. I approve. However, I'm concerned about how PATCH and DELETE do not say 'stubbed' in Cypress. Not sure if this is just an anomaly or not.

Screen Shot 2021-07-19 at 10 21 10

**Pull request merged**